translate/lang/data.py

```diff
@@ -239,6 +239,75 @@
 }
 
 
+scripts = {
+    'devanagari': [
```

Comment:
Since we can make these be any name.  I'd suggest using [ISO 15924](http://unicode.org/iso15924/iso15924-codes.html) so that we can easily follow the standard.